### PR TITLE
Adapt api.MediaKeySession.onmessage to new events structure

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adapts the message event of the MediaKeySession API to conform to the new events structure.

### CODE REVIEW

**Review by foolip — CHANGES_REQUESTED** [review]
https://source.chromium.org/chromium/chromium/src/+/6e3647df59a9b8a2c77d49ce5681e2a5b3ab7d17 shows that the event handler attributes were added after the events themselves. Can you add `partial_implementation` ranges?

**Comment by queengooborg** [comment]
I actually don't know how to track down support for the events themselves in the source code!  Would you be willing/able to provide me some assistance there?

**Comment by Elchi3** [comment]
I think the following is needed here:

```
"chrome": [
  {
    "version_added": "55"
  },
  {
    "version_added": "42",
    "version_removed": "55",
    "partial_implementation": true,
    "notes": "The <code>onkeystatuseschange</code> event handler property is not supported."
  }
],
```

And 

```
"chrome": [
  {
    "version_added": "55"
  },
  {
    "version_added": "42",
    "version_removed": "55",
    "partial_implementation": true,
    "notes": "The <code>onmessage</code> event handler property is not supported."
  }
],
```

(same structure [with different versions] for other chromium based browsers)

File: api/MediaKeySession.json [hunk 1]
```diff
@@ -348,12 +348,28 @@
...
           "support": {
-            "chrome": {
-              "version_added": "55"
-            },
-            "chrome_android": {
-              "version_added": "55"
-            },
+            "chrome": [
+              {
+                "version_added": "55"
```
Comment (foolip) [inline api/MediaKeySession.json:353]: Confirmed by https://storage.googleapis.com/chromium-find-releases-static/6e3.html#6e3647df59a9b8a2c77d49ce5681e2a5b3ab7d17

**Review by foolip — APPROVED** [review]

**Review by foolip — APPROVED** [review]

**Pull request merged**